.github/renovate.json

```diff
@@ -10,43 +10,19 @@
   ],
```

Comment:
Since it's currently scheduled to only run monthly, we should trigger a manual run after this PR completes (can do from #1537) so that we don't wait a month to discover any issues with this approach.

Reply:
Makes sense, can close the existing PRs, clear the ignore lists (found the UI yesterday!), and run again